deploy/openshift/operator.yaml

```diff
@@ -40,13 +59,16 @@ spec:
         volumeMounts:
         - name: dvo-config
           mountPath: /config
+        securityContext:
+          runAsUser: 1000
```

Comment:
We don't want to do a runAsUser.  This requires elevated privs on openshift.

Reply:
Hm, so you propose we just remove `runAsUser` from this and leave it at that? 

I'm okay with that, although DVO will then still complain about the `run_as_non_root` validation on itself.

Do you think we should add `run_as_non_root` to the [incompatible checks list](https://github.com/app-sre/deployment-validation-operator/blob/47264bae4f9a1f9514f5f8423082e6c0637219ab/pkg/validations/validation_engine.go#L178-L184)?

Reply:
No, run_as_non_root is fine for a kubernetes environment.  For openshift however, pods are run as non-root by default because they are run as a random UID.  Using runAsUser requires elevated privs either way, but will be problematic on openshift w/o any benefit.

Reply:
For now let's not add the runAsUser.

Reply:
It has been removed.